devex/vscode-extension/assets/baseSolutionTemplateFolder/enc/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
See earlier comment about replacing all occurrences of this line.

Reply:
Idem.